build_darwin.sh

```diff
@@ -13,7 +13,7 @@ cd ..
 curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$LIBEVENT_VERSION/libevent-$LIBEVENT_VERSION.tar.gz" -o libevent-$LIBEVENT_VERSION.tar.gz && \
 curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$LIBEVENT_VERSION/libevent-$LIBEVENT_VERSION.tar.gz.asc" -o libevent-$LIBEVENT_VERSION.tar.gz.asc && \
 
-gpg --keyserver pool.sks-keyservers.net/ --recv-keys $LIBEVENT_KEY && \
+gpg --keyserver $KEYSERVER --recv-keys $LIBEVENT_KEY && \
```

Comment:
double-quote (both of these, out of paranoia, even though the key id is not likely to have shell metacharacters like spaces in it)